CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Added support for the Steam authentication flow.
 
+### Changed
```

Comment:
Arguably, this could be `fixed` as well 🤷‍♂️. Not sure which is more correct though